src/crypto/CryptoTests.cpp

```diff
@@ -256,7 +256,7 @@ TEST_CASE("StrKey tests", "[crypto]")
 
         for (size_t p = 0u; p < encoded.size(); p++)
         {
-            for (int st = 0; st < 4; st++)
+            for (int st = 0; st < 9; st++)
```

Comment:
Why 9 here? Was the intention for `3 <= st < 9` to all have the same outcome?

Reply:
I ended up rewriting what I had, new version is easier to understand